packages/button/src/ux-button.ts

```diff
@@ -1,10 +1,12 @@
-import { customElement, bindable } from 'aurelia-templating';
+import { customElement, bindable, inlineView } from 'aurelia-templating';
 import { inject } from 'aurelia-dependency-injection';
 import { StyleEngine, UxComponent, PaperRipple, normalizeBooleanAttribute } from '@aurelia-ux/core';
 import { UxButtonTheme } from './ux-button-theme';
+import UX_BUTTON_VIEW from './ux-button.html';
```

Comment:
Wouldn't we want the casing to be `UxButtonView` instead of `UX_BUTTON_VIEW` to be consistent?

Reply:
I was not sure about this, so I named it like a constant. Wouldn't be an issue if you want to proceed.